server/review/api.ts

```diff
@@ -15,9 +15,11 @@ const getRequestIds = (req) => {
 
 app.post('/api/reviews', (req, res) => {
 	const requestIds = getRequestIds(req);
+	const { permissions } = req.body;
 	getPermissions(requestIds)
-		.then((permissions) => {
-			if (!permissions.create) {
+		// how should permissions exist on a api call that can be used by anyone?
```

Comment:
Dunno!
I defer on resolving this to press on with the rest of the review, but don't want it to slip through.